src/WebJobs.Script.WebHost/Middleware/HostWarmupMiddleware.cs

```diff
@@ -65,11 +79,22 @@ public async Task WarmupInvoke(HttpContext httpContext)
 
             ReadRuntimeAssemblyFiles();
 
-            await WarmUp(httpContext.Request);
+            await HostWarmup(httpContext.Request);
+
+            await WorkerWarmup();
 
             await _next.Invoke(httpContext);
         }
 
+        private async Task WorkerWarmup()
+        {
+            if (_hostingConfigOptions.Value.WorkerWarmupEnabled)
+            {
+                _logger.LogDebug("Worker warmup hosting configuration is enabled");
```

Comment:
nit: no need to log this as this is going to be on by default and is an excessive log for each warmup request.